### ISSUE TITLE
When adding XP drops to a block, XP ball appears but does not add to bar or make ding noise

### ISSUE THREAD

**Issue by Luke616** [codetaylor/dropt#13]
Title, here is one of the scipts I am attempting to run. https://pastebin.com/Zk4cwdFG

When this block is mined, 1-2 ore items drops as expected, along with 1 XP ball. However, the XP ball does nothing when picked up - it does not increase XP count, fill the bar, or make a ding sound.

Minecraft 1.12.2
Forge 14.23.2.2623
Dropt 1.6.9
Athaenaeum 1.13.0

**Comment by codetaylor** [comment]
Thanks for the report, I'm looking into it now.

**Comment by codetaylor** [comment]
Got it fixed!

Also, when using something like this:

```json
{
  "min": 1,
  "max": 1,
  "fortuneModifier": 0
}
```

You can do this instead:

```json
{
  "fixed": 1,
  "fortuneModifier": 0
}
```

Or just this:

```json
{
  "fixed": 1
}
```

It does the same thing. :)

**Comment by codetaylor** [comment]
Fixed in 1.12.2-1.6.10

**Comment by Luke616** [comment]
Thanks so much for the quick fix, and for the tip!
